Automattic/wp-calypso
# Onboarding: Give your blog a name step

#### Changes proposed in this Pull Request

* Implement “Give your blog a name” step based on the horizontal layout introduced by https://github.com/Automattic/wp-calypso/pull/56165
* Note that this PR focuses on “Give your blog a name” step, so there might be some issue when you go back to select another intent.

##### Demo

https://user-images.githubusercontent.com/13596067/135252268-3334b367-3822-4f5b-aca6-03e2d8398236.mov

##### Screenshots

Desktop

<img src="https://user-images.githubusercontent.com/13596067/135079783-15684d44-4b56-4041-b8f6-f139fc0411ae.png" />

Mobile

<img src="https://user-images.githubusercontent.com/13596067/135079810-5468c5e8-c360-47e2-8249-e24660a9a048.png" />

#### Testing instructions



* Go to `/start/setup-site?flags=signup/hero-flow&siteSlug=<your_site>`
* Select “write” intent and complete giving your blog a name step
* Check redirecting editor or not after completing all of steps
* Go to My Home > Settings > General and check blog name and tagline are correct.



Related to https://github.com/Automattic/wp-calypso/issues/55603
